src/FamilyHubs.ServiceDirectory.Api/Endpoints/MinimalTaxonomyEndPoints.cs

```diff
@@ -16,7 +13,7 @@ public class MinimalTaxonomyEndPoints
 {
     public void RegisterTaxonomyEndPoints(WebApplication app)
     {
-        app.MapPost("api/taxonomies", [Authorize(Policy = "ServiceAccess")] async ([FromBody] TaxonomyDto request, CancellationToken cancellationToken, ISender mediator, ILogger<MinimalTaxonomyEndPoints> logger) =>
+        app.MapPost("api/taxonomies", async ([FromBody] TaxonomyDto request, CancellationToken cancellationToken, ISender mediator, ILogger<MinimalTaxonomyEndPoints> logger) =>
```

Comment:
Shouldn't Post / Put operations  have some sort of Authorize policy for all endpoints?